build_runner/lib/src/package_graph/package_graph.dart

```diff
@@ -133,6 +135,7 @@ class PackageGraph {
   ///
   /// See [orderedPackages] for ordering guarantees. The node for [packageName]
   /// will not be included in the result.
+  @Deprecated('Use `applyBuilders` to filter `orderedPackageCycles`')
```

Comment:
Mention deprecations in the changelog...

Reply:
Done